lib/services/messenger/translators/discourse.ts

```diff
@@ -328,12 +347,33 @@ export class DiscourseTranslator extends TranslatorScaffold implements Translato
 		return Promise.resolve({});
 	}
 
+	private static convertReadThreadResponse(
+		config: MetadataConfiguration,
+		show: 'reply' | 'whisper' | 'all',
+		_message: TransmitInformation,
+		response: any,
+	): Promise<TranslatorMetadata[]> {
+		const messages = _.compact(_.map(response.post_stream.posts, (comment) => {
```

Comment:
May as well just:
```typescript
return Promise.resolve(_.compact...);
```
I don't think it's messy, given it's a single function within the resolve.